src/attestation/Attestation.ts

```diff
@@ -21,6 +21,20 @@ import IPublicIdentity from '../types/PublicIdentity'
 
 const log = factory.getLogger('Attestation')
 
+export function compressAttestation(attestation: IAttestation): any[] {
+  return Object.values(attestation)
```

Comment:
not sure if this is a good idea. We are relying on a fixed order. Does Object.values provide a fixed oder?

those two things are not the same(?):

```js
Object.values({"b":1, "a":2})
Object.values({"a":2, "b":1})
```

Reply:
Yes, this is one of the main things I have to deal with. Currently, we have proposed jsonabc, which can sort. 

So, this will be the main thing, taking unsorted JSON objects and sorting them into our current order. E.g. what is suitable for the specific type.